### ISSUE TITLE
Accessibility: Missing Form Control for Show x entries on Tables

### ISSUE THREAD

**Issue by yousoph** [apache/superset#30669]
## Screenshot

![image](https://github.com/user-attachments/assets/10258645-b014-4aec-82b6-99c17007dd2e)

## Description

To get to this state: 
1. Create a table chart
2. Enable server pagination 
3. Save the chart and add it to a dashboard 

Currently, the "Show x entries" doesn't have a label when using a screen reader

**Comment by geido** [comment]
---

### 🎉 **Preset Bounty Available: $50 USD** 🎉

To claim this bounty, please carefully follow the steps below.

---

#### 📋 **Steps to Participate**

1. **Review Guidelines:**  
   Read through the [Preset Bounty Program Contribution Guide](https://preset.notion.site/preset-bounty-program-apache-superset) for complete details on bounty requirements.

2. **Show Your Interest:**  
   Complete the [Preset Bounty Program Survey](https://docs.google.com/forms/d/e/1FAIpQLSeDsR1iWvPxFydUIXs2agsUg3kMDaAooeD4P9P1L5sI0Yu5QQ/viewform?usp=sharing) and comment this issue to express your interest.

3. **Join the Slack Channel:**  
   After completing the survey, you’ll receive an invitation to the dedicated [Apache Superset Slack](https://join.slack.com/t/apache-superset/shared_invite/zt-2be0drwz8-bxPfkdz28ozzk1Iox29ufg) channel.

4. **Get Assigned:**  
   To officially start, ensure a **Bounty Program Manager** has assigned you to this issue.

5. **Submit Your Solution:**  
   When ready, submit your solution with the `Fixes #{issue_number}` notation in your Pull Request description.

6. **Claim Your Bounty:**  
   Sign up at [GitPay.me](https://gitpay.me/) and submit your solution via: [https://gitpay.me/#/task/967](https://gitpay.me/#/task/967)

---

#### 💡 **Additional Notes**

- Only developers assigned by a **Bounty Program Manager** should start working on this issue to win the bounty.
- Be sure to follow the guide closely to avoid any delays in payment. Please, allow a few days after your PR has been merged for the bounty to be released.

Good luck, and happy coding! 🎉

**Comment by sachinks07** [comment]
I would like to work on this issue. Thanks

**Comment by LevisNgigi** [comment]
Hello I have filled the survey form and would love to work on this issue.

**Comment by aybanda** [comment]
Hey @geido would like work on this issue, can you please assign this to me as I have delivered similar works before.

**Comment by sachinks07** [comment]
Hi @geido , I commented first on the issue but wasn’t assigned. I’m curious about the decision. Thanks!

**Comment by geido** [comment]
> Hi @geido , I commented first on the issue but wasn’t assigned. I’m curious about the decision. Thanks!

The decision is not based on who comments first but it is based on the survey submissions, individual skills (based on previous contributions) and other factors. Thanks for your interest though. There will be more opportunities soon.
